go.mod

```diff
@@ -3,57 +3,74 @@ module github.com/open-telemetry/opentelemetry-collector
 go 1.13
```

Comment:
Do we also need to update this if we require go 1.14?

Reply:
Yes, I believe this is a good idea to prevent people from accidentally regressing go.mod to the wrong `+incompatible` version by running `go mod tidy` or `go mod get` with an older golang.